### ISSUE TITLE
Support for unknown dynamic route

### ISSUE THREAD

**Issue by nonsocode** [ktsn/vue-route-generator#156]
There's currently no support for a 'Catch all' route similar to what's available as seen [in Nuxt](https://nuxtjs.org/guide/routing/#unknown-dynamic-nested-routes)

It'll be nice to add support for `_.vue` files to be a catchall route.
```
pages/
--| people/
-----| _id.vue
-----| index.vue
--| _.vue
--| index.vue
```

**Comment by ktsn** [comment]
Sounds like we should implement it in this lib as well. btw, you can do that by just passing normal vue-router config in the current version.

```ts
import Vue from 'vue'
import Router from 'vue-router'
import routes from './routes'

Vue.use(Router)

export default new Router({
  routes: [
    ...routes,
    
    {
      path: '*',
      redirect: '...'
    }
  ]
})
```

**Comment by nonsocode** [comment]
@ktsn Agreed. But the underscore notation allows for more consistency with the convention and would be very useful for nested routes
